spec/lib/replica_spec.rb

```diff
@@ -178,6 +178,21 @@
         expect(response.count).to eq(28)
       end
     end
+
+    it 'functions identically on wikimedia incubator' do
+      VCR.use_cassette 'replica/wikimedia_incubator_revisions' do
+        all_users = [
+          build(:user, username: 'Ragesoss')
```

Comment:
This should confirm that we can actually get results back. Choose a user and date change that return some revisions. For example: http://tools.wmflabs.org/wikiedudashboard/revisions.php?lang=incubator&project=wikipedia&usernames[]=Daad%20Ikram&start=20170311000000&end=20170317000000

Username: `Daad Ikram`

dates: 20170311000000 to 20170317000000